pkg/csi/service/wcp/controller.go

```diff
@@ -641,8 +627,9 @@ func (c *controller) createFileVolume(ctx context.Context, req *csi.CreateVolume
 			"no datastores found to create file volume")
 	}
 	filterSuspendedDatastores := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.CnsMgrSuspendCreateVolume)
+	useSupervisorId := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA)
```

Comment:
would be better to name it as `isTKGSHAEnabled`?

Also, would it be better to pass a func rather than a value here? This way, when the `isTKGSHAEnabled` func is called, the FSS state in the real-time will be used. Instead, passing a value might end up with a stale  one.

Reply:
used `isTKGSHAEnabled` instead of `useSupervisorId`.
it is ok to rely on the stale value of `isTKGSHAEnabled` as we do not support runtime feature state enablement. checking feature state requires checking config-map and trip to the API server so using cached value.